include/zephyr/acpi/acpi.h

```diff
@@ -88,6 +88,7 @@ struct acpi_mmio_resource {
  */
 #define ACPI_DT_HID(node_id) DT_PROP_OR(node_id, hardware_id, "INVALD")
 
+
```

Comment:
Something is wrong with this commit. It isn't doing anything related to the commit description.

Reply:
this is due to dependency PR https://github.com/zephyrproject-rtos/zephyr/pull/62694 and rebased it